lib/components/narrative/tabbed-itineraries.js

```diff
@@ -88,9 +89,11 @@ class TabButton extends Component {
     )
     return (
       <Button
+        ariaRole="tab"
         className={classNames.join(' ')}
         key={`tab-button-${index}`}
         onClick={this._onClick}
+        tabIndex={isActive ? '0' : '-1'}
```

Comment:
This would make inactive tabs unreachable from the tab key, right? Do we want to set the value to zero for all of them?

Reply:
I'm not sure... I followed https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example

Reply:
Okay. Looks like the parent element needs a `role="tablist"` per your link above.